src/styles/sidebar/components/tag-editor.scss

```diff
@@ -28,7 +27,7 @@
     border: 1px solid var.$grey-3;
     border-radius: 2px 0 0 2px;
     border-right-width: 0;
-    padding: 0 0.5em;
+    padding: 2px 0.5em;
```

Comment:
This deserves to be revisited but was formerly achieved by applying a custom `line-height` here.